app/src/main/java/com/codesoom/assignment/models/HttpStatus.java

```diff
@@ -0,0 +1,15 @@
+package com.codesoom.assignment.models;
+
+public enum HttpStatus {
+    HTTP_OK(200), HTTP_CREATE(201), HTTP_NO_CONTENT(204), HTTP_NOT_FOUND(404);
```

Comment:
enum의 이름이 있으니 앞의 `HTTP`는 생략해도 좋아 보입니다. 한편 `HTTP_CREATE(201)`는 HTTP 공식 호칭이 `CREATE`가 아니라 `CREATED` 이므로 `CREATED`를 쓰는 것이 바람직하겠죠.

```suggestion
    OK(200), CREATED(201), NO_CONTENT(204), NOT_FOUND(404);
```

이런 이름을 지을 때에는 공식 문서를 읽고, 공식 문서의 이름을 그대로 사용하는 습관을 들이면 좋습니다.

어떤 문서를 읽어야 할 지 모르겠다면 RFC 2616을 읽어보세요.
https://datatracker.ietf.org/doc/html/rfc2616

이 문서는 1999년에 HTTP의 창시자인 팀 버너스 리와 REST 창시자인 로이 필딩 등이 함께 작성한 HTTP 1.1의 스펙 문서입니다.
즉 이 문서가 HTTP와 관련된 가장 중요한 문서라고 할 수 있어요. 틈날 때마다 읽어두시면 큰 도움이 될 거에요.
백엔드이건 프론트이건 HTTP 공식 문서의 존재를 알아두는 건 엄청 중요한 일입니다.